examples/RED/main.go

```diff
@@ -26,15 +25,15 @@ func doTheWork(wtb workTimeBox) {
 func main() {
 	var err error
 
-	reporter, _ := strategy.NewRED(strategy.REDOpts{
+	redExample, _ := strategy.NewRED(strategy.REDOpts{
```

Comment:
Error handling? I guess you don't really need it for an example but I still think it's nice to have.